### ISSUE TITLE
click is no use in linux/win7

### ISSUE THREAD

**Issue by Linbubin** [wulkano/kap#211]
I fork the progrem. and changed some code so that it can run under linux.

but when i click the `<header class="kap-header">....</header>` part, there was no response.
the `<section class="controls">....</section>` part is response.

who can tell me the reason? 

the progrem in [this](https://github.com/Linbubin/kap), you can see the recent commit to find what i remove or change.
and run 

```
npm install
cd app && npm install
cd ..
npm run build && npm start
```
to run the program.

```
os : Linux debian 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt25-2+deb8u3 (2016-07-02) x86_64 GNU/Linux
npm -v : 5.2.0
electron -v : v1.6.11
```
anyway, thanks.

**Comment by kevva** [comment]
You have removed all calls to `aperture` which is responsible for the screen recording functionality, so this won't work anyway.

**Comment by skllcrn** [comment]
Thanks @kevva, I also suggest looking into #31 and https://github.com/wulkano/aperture/pull/32

**Comment by Linbubin** [comment]
@kevva @skllcrn  I don't need to implement the recording function.
 I mean, click events are not use.

**Comment by matheuss** [comment]
If you're on macOS and `-webkit-app-region: drag;` is applied to an element, such element will receive click events normally, despite the [docs saying the opposite](https://github.com/electron/electron/blob/master/docs/api/frameless-window.md#draggable-region).

On Linux and Windows, however, the element will **not** receive click events, like mentioned in the docs.

TLDR: remove this [two lines](https://github.com/wulkano/kap/blob/master/app/src/renderer/css/_helpers.css#L11-L12)

**Comment by Linbubin** [comment]
@matheuss  cool,it useful.
thanks very much.
This problem has troubled me for a long time. :smile:
